### PR TITLE
translated category names into spanish (condensed previous small PR'S)

### PR DESCRIPTION
**Pull request by dr-nefardio3058**

### CODE REVIEW

**Comment by dr-nefardio3058** [comment]
@trigaten done, close other PR'S

File: docs/bibliography.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 sidebar_position: 1000
```
Comment (trigaten) [inline docs/bibliography.md:2]: This is translating the English version

File: docs/intermediate/_category_.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "label": "🧙‍♂️ Intermediate",
```
Comment (trigaten) [inline docs/intermediate/_category_.json:2]: same here

**Comment by dr-nefardio3058** [comment]
I'll look at the suggestions when I find time. Thanks for reviewing.

**Comment by dr-nefardio3058** [comment]
Resolved all suggested chaneges in the last commit of this pull request

File: i18n/es/docusaurus-plugin-content-docs/current/Images/_category_.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "label": "🖼️ Image Prompting",
+  "label": "🖼️ Parámetros para imágenes",
```
Comment (trigaten) [inline i18n/es/docusaurus-plugin-content-docs/current/Images/_category_.json:2]: I think we should keep the word prompt here

File: i18n/es/docusaurus-plugin-content-docs/current/intermediate/zero_shot_cot.md [hunk 4]
```diff
@@ -2,8 +2,7 @@
 sidebar_position: 4
 ---
 
-# 🟢 Zero Shot Chain of Thought
-
+# 🟢 Cadena de pensamiento sin entrenamiento previo
```
Comment (trigaten) [inline i18n/es/docusaurus-plugin-content-docs/current/intermediate/zero_shot_cot.md:5]: Why is `sin entrenamiento previo` necessary?

File: i18n/es/docusaurus-plugin-content-docs/current/Images/_category_.json (hunk 3 above)
Comment (dr-nefardio3058) [reply i18n/es/docusaurus-plugin-content-docs/current/Images/_category_.json:2]: I'll leave 'image prompting' then. Appreciate the suggestion.

File: i18n/es/docusaurus-plugin-content-docs/current/intermediate/zero_shot_cot.md (hunk 4 above)
Comment (dr-nefardio3058) [reply i18n/es/docusaurus-plugin-content-docs/current/intermediate/zero_shot_cot.md:5]: Context. Text below the H1 is "La cadena de pensamiento sin entrenamiento previo (Zero Shot Chain of Thought, Zero-shot-CoT"

**Pull request closed without merging**